packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -1,5 +1,8 @@
-import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { Component, Element, Host, Prop, State, h } from '@stencil/core';
 import { isNumeric } from '../../utils/utils';
+import ascendingIcon from '../../assets/sort-arrow-up.svg?format=text';
+import descendingIcon from '../../assets/sort-arrow-down.svg?format=text';
```

Comment:
I'd like to explore moving this kind of thing into a `<va-icon>` component in the future, where a prop can be used to identify the icon name for a dynamic import.

Also, the `?format=text` was the only way I was able to import the raw svg instead of the encoded string.